backend/init_database.js

```diff
@@ -0,0 +1,58 @@
+const { debug } = require("console");
+const fs = require("fs");
+
+function addPoem (poem) {
+    const db = require('./database.json')
+    const newPoem = poem
+    db.push(newPoem)
+    fs.writeFile('database.json', JSON.stringify(db), err => {
```

Comment:
in order for the test script to work we're going to want to replace `'databas.json'` here with the new `dbpath` variable that has the absolute path. also, in order to make our database a bit more legible let's do `JSON.stringify(db, null, 2)` which will add line breaks to our objects && indent our database w/2 spaces per tab (rather than writting it all on one line)